examples/multi_gpu/papers100m_gcn.py

```diff
@@ -1,142 +1,331 @@
+import argparse
 import os
+import tempfile
 import time
 
+import numpy as np
 import torch
 import torch.distributed as dist
 import torch.multiprocessing as mp
 import torch.nn.functional as F
 from ogb.nodeproppred import PygNodePropPredDataset
+from torch.distributed.algorithms.join import Join
 from torch.nn.parallel import DistributedDataParallel
+from torchmetrics import Accuracy
 
-from torch_geometric.loader import NeighborLoader
-from torch_geometric.nn import GCNConv
+import torch_geometric
 
+os.environ['CUDF_SPILL'] = '1'
+os.environ['RAPIDS_NO_INITIALIZE'] = '1'
```

Comment:
It'd also be nice to add some comment here as well. Is this for suppressing warnings in the cluster?

Reply:
@alexbarghi-nv
will provide the answer for this since im not really sure personally. (i also dont know much about dask_cuda😅)

Reply:
CUDF_SPILL enables cudf spilling which saves device memory.  RAPIDS_NO_INITIALIZE ensures that a CUDA context is not created on import of rapids, allowing pytorch to create the context instead.